staff/fede-gomez/js-ayay/ayay.spec.js

```diff
@@ -0,0 +1,134 @@
+describe('Ayay', function () {
+    var ayay;
+
+    beforeEach(function() {
+        ayay = new Ayay;
+    });
+
+    describe('push', function () {
+        it('should push items', function () {
+
+            var numOfAddedElems = 20;
+            for(var i=0; i<numOfAddedElems; i++) {
+                ayay.push(i+1);
+            }
+
+            expect(ayay.length).toEqual(numOfAddedElems);
+
+            for (var i = 0; i < ayay.length; i++)
+                expect(ayay[i]).toEqual(i+1);
+        });
+        
+        it('should return the length of the ayay even if no element is passed as argument', function () {
```

Comment:
you could check other things as if it's possible to push chars, or if it's possible to push more than one item at once